### PR TITLE
Swapped png build status badge for svg version

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by dawsonbotsford** [comment]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# image-diff [![Build status](https://travis-ci.org/uber/image-diff.png?branch=master)](https://travis-ci.org/uber/image-diff)
+# image-diff
+[![Build Status](https://travis-ci.org/uber/image-diff.svg?branch=master)](https://travis-ci.org/uber/image-diff)
```
Comment (twolfson) [inline README.md]: Could we keep these on the same line? It's a stylistic preference =/

**Comment by amilajack** [comment]
Sure

**Comment by twolfson** [comment]
Looks good to me. Notes for release process:

Patch version release should be fine. Here's some notes for the `CHANGELOG`:

```md
1.6.2 - Updated Travis CI badge from PNG to SVG via @amilajack in #44
```

Commands for new release:

```bash
./release.sh 1.6.2
```

/cc @mlmorg

**Comment by twolfson** [comment]
Thanks @mlmorg! It looks like we forgot to update the CHANGELOG though =/

**Comment by mlmorg** [comment]
Oops... updating now.

**Pull request merged**